buildspec/windowsTests.yml

```diff
@@ -15,8 +15,8 @@ phases:
   install:
     commands:
       - choco install -y --no-progress codecov
-      - choco install -y --no-progress dotnetcore-sdk --version 2.2.104
-      - choco install -y --no-progress netfx-4.6.1-devpack
+    runtime-versions:
```

Comment:
can we use this in the linux builds as well instead of manually installing stuff?

Reply:
looks like we do: https://github.com/aws/aws-toolkit-jetbrains/blob/3cb1f17f301d0fed5dd2f1732f4f16b2c2b0b7d2/buildspec/linuxUiTests.yml#L17

@abrooksv mentioned that we aren't running Rider tests on CI yet, so I'm not sure if this PR is ok. Do we plan to enable the rider tests on CI?